pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "script-languages"
+version = "0.1.0"
+description = "Script Languages"
+
+license = "MIT"
+
+authors = [
+    "Torsten Kilias <torsten.kilias@exasol.com>"
+]
+
+readme = 'README.md'  # Markdown files are supported
+
+repository = "https://github.com/exasol/script-languages"
+homepage = "https://github.com/exasol/script-languages"
+
+keywords = ['exasol', 'udf', 'script-languages']
+
+[tool.poetry.dependencies]
+python = ">=3.8,<4"
+importlib-resources = ">=5.4.0"
+exasol_script_languages_container_tool = { url = "https://github.com/exasol/script-languages-container-tool/releases/download/0.12.0/exasol_script_languages_container_tool-0.12.0-py3-none-any.whl" }
+exasol-script-languages-container-ci = {git = "https://github.com/exasol/script-languages-container-ci.git", branch="feature/9_implement_run-release_command"}
+exasol-script-languages-container-ci-setup = {git = "https://github.com/exasol/script-languages-container-ci-setup.git", branch="feature/3_addess_release_code_build_and_command"}
+#exasol-script-languages-container-ci = { path = "/home/thomas/Work/script-languages-container-ci", develop = false}
```

Comment:
Cleanup

Reply:
Once this PR and https://github.com/exasol/script-languages-container-ci-setup/pull/5 is approved I will create releases for exasol-script-languages-container-ci and exasol-script-languages-container-ci-setup and then update the dependencies here accordingly.